amethyst_utils/src/render.rs

```diff
@@ -0,0 +1,45 @@
+//! Module containing utility methods to easily create `Material` and `Texture` handles using a minimal amount of code.
+
+use amethyst_assets::{AssetStorage, Handle, Loader, Progress};
+use amethyst_core::specs::{Read, ReadExpect};
+use amethyst_renderer::{Material, MaterialDefaults, PngFormat, Texture, TextureMetadata};
+
+#[derive(SystemData)]
+pub struct MaterialCreator<'a> {
+    loader: ReadExpect<'a, Loader>,
+    storage: Read<'a, AssetStorage<Texture>>,
+    defaults: ReadExpect<'a, MaterialDefaults>,
+}
+
+impl<'a> MaterialCreator<'a> {
+    /// Generate a `Material` from a color.
+    pub fn material_from_color<T: Progress>(
```

Comment:
Let's call this `material_from_albedo`

Reply:
Technically its material_from_albedo_from_color. Are you sure you want it called albedo?

Reply:
Dismissing...